frontend/packages/topology/src/components/list-view/TopologyListView.scss

```diff
@@ -12,7 +12,6 @@
   right: 0;
   overflow-x: hidden;
   overflow-y: auto;
-  padding-left: var(--pf-global--spacer--sm);
```

Comment:
This change results that the selected nodes in the topology list doesn't have a spacing on the left side anymore.

Before:

![image](https://user-images.githubusercontent.com/139310/164285335-1cda7fef-1810-4abc-b656-d814626ccf7c.png)

With the current version:

![image](https://user-images.githubusercontent.com/139310/164285356-a548cab1-2b43-4237-87f1-0cd3932ad9d9.png)

I would suggest that you don't remove this line. But then the inner data list are not aligned with the rest of your changes!

You can remove this `padding-left` by adding this rule below `.odc-topology-list-view`:

```css
  // Removes the padding-left from above from the inner grid padding-left so that the
  // project dropdown, add to project button and the grid content are aligned.
  // See also https://github.com/patternfly/patternfly/blob/prerelease-v4.185.1/src/patternfly/components/DataList/data-list-grid.scss#L37
  .pf-c-data-list {
    --pf-c-data-list__item-row--PaddingLeft: calc(var(--pf-c-data-list__item-row--xl--PaddingLeft) - var(--pf-global--spacer--sm));
  }
```

With this change it looks like this:

![image](https://user-images.githubusercontent.com/139310/164285633-6683ea73-08b5-4c18-9bfe-f6c03c4e4f3a.png)

Wdyt? It still a UI change which we should check with the UX team.

@jeff-phillips-18 @beaumorley Is this slightly changed topology layout fine for you as well??

Reply:
First screenshot above is from 4.9.

4.10 and 4.11 the text is misaligned like this
![Screenshot from 2022-04-20 19-16-44](https://user-images.githubusercontent.com/139310/164286845-db336e2c-c42f-4e8a-9660-000beee529f9.png)

And Avik wants to align all these things again.

Reply:
I'd agree with Avik

Reply:
Thanks, @jeff-phillips-18 @jerolimov 
I have updated the code to fix that CSS issue